exercises/practice/book-store/.meta/example.cpp

```diff
@@ -0,0 +1,66 @@
+#include <array>
+#include <limits>
+#include <map>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include "book_store.h"
+
+namespace book_store {
+
+static const int PRICE_AFTER_DISCOUNT[6] = {0, 800, 1520, 2160, 2560, 3000};
```

Comment:
This can be `constexpr`.

Reply:
Probably would make sense to use `std::array` here. I can imagine all template parameters could be automatically derived, i.e.:
```suggestion
static constexpr std::array price_after_discount = {0, 800, 1520, 2160, 2560, 3000};
```

It looks like we're not quite consistent with uppercase vs lowercase for constants, but I'd be very much in favor of using `ALL_CAPS` only for preprocessor macros. @vaeng Do we have anything official on this?

Reply:
Both done. Are there any ressources for coding styles for this repo? I don't want to create extra work

Reply:
Yes, screaming snake makros. I would adhere to the C++ Core Guidelines. (Which I might not have in some places where it slipped my mind) https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-all-caps